src/test/java/org/apache/maven/shared/utils/io/SymlinkTestSetup.java

```diff
@@ -36,18 +37,18 @@ public static File createStandardSymlinkTestDir( File root )
         srcDir.mkdirs();
         File target = new File( srcDir, "targetDir" );
         target.mkdirs();
-        write( new File( target, "targetFile.txt" ), "a regular File payload" );
+        write( new File( target, "targetFile.txt" ), "a regular File payload", StandardCharsets.UTF_8 );
         File aRegularDir = new File( srcDir, "aRegularDir" );
         aRegularDir.mkdirs();
-        write( new File( aRegularDir, "aRegularFile.txt" ), "a regular File payload" );
+        write( new File( aRegularDir, "aRegularFile.txt" ), "a regular File payload", StandardCharsets.UTF_8 );
 
         File dirOnTheOutside = new File( root, "dirOnTheOutside" );
         dirOnTheOutside.mkdirs();
-        write( new File( dirOnTheOutside, "FileInDirOnTheOutside.txt" ), "a file in dir on the outside" );
-        write( new File( root, "onTheOutside.txt" ), "A file on the outside" );
-        write( new File( srcDir, "fileR.txt" ), "FileR payload" );
-        write( new File( srcDir, "fileW.txt" ), "FileW payload" );
-        write( new File( srcDir, "fileX.txt" ), "FileX payload" );
+        write( new File( dirOnTheOutside, "FileInDirOnTheOutside.txt" ), "a file in dir on the outside", StandardCharsets.UTF_8 );
```

Comment:
Looking at this, if you need to change the method call anyway, I'd rewrite it using https://docs.oracle.com/javase/8/docs/api/java/nio/file/Files.html#write-java.nio.file.Path-java.lang.Iterable-java.nio.file.OpenOption...-

Reply:
That method's new in Java 1.8, and we want to support Java 1.7.

Reply:
Ah, I see, so leave it like this.